src/browser/components/ShowMoreOrAll/ShowMoreOrAll.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react'
+import { StyledShowMoreLink } from './styled'
+
+type ShowMoreOrAllProps = {
+  total: number
+  shown: number
+  moreStep: number
+  onMore: (num: number) => any
+}
+export const ShowMoreOrAll = ({
+  total,
+  shown,
+  moreStep,
+  onMore
+}: ShowMoreOrAllProps) => {
+  const numMore = total - shown > moreStep ? moreStep : total - shown
```

Comment:
This is just my personal preference, so no need to change anything: I usually find Math.min easier to understand for these kinds of things, like so: 
```javascript
const numMove = Math.min(moreStep, total - shown) 
```

Reply:
yes, agree that it is more readable, will change!